src/ElvClient.js

```diff
@@ -738,13 +738,16 @@ class ElvClient {
     };
 
     if(!Sign) {
-      Sign = async message => this.authClient.Sign(message);
+      Sign = async message => this.authClient.Sign(message, !addEthereumPrefix);
     }
 
-    let message = `Eluvio Content Fabric Access Token 1.0\n${JSON.stringify(token)}`;
```

Comment:
Don't make this prefix conditional, it is part of our token spec. The `addEthereumPrefix` here is optional because the sign method may be metamask, which already adds the prefix.

Reply:
Okay, I will have to add a big more logic since the authd wallet API for personal_sign also adds the custom eluvio prefix

Reply:
that... seems wrong. this prefix is specific to this type of token, but ethereum prefixed sign is a standard. @elv-serban?

Reply:
I will defer to Serban then; that is what I was originally told to leave in the API

Reply:
Serban confirmed that the Eluvio prefix should only be added client side here, not server side, so I will adjust accordingly